### ISSUE TITLE
Ex. 8 Constraints

### ISSUE THREAD

**Issue by bazzif** [CMM-22/a6#10]
I wonder what you mean by: 
Add a |-|^2 regularizer on the vector of optimisation variables [u x v ]^T.
Does that mean we should calculate the square of the Frobenius norm of the stack vector?

**Comment by MiguelZamoraM** [comment]
Yes. That's the easiest way to add some regularization.  You could also split the optimization variables and compute the squared norm for the control and states variables separately. That way you could also weigh the regularization of the controls and the state variables using different coefficients, but the squared of the entire vector should also be ok.
